cmd/main.go

```diff
@@ -39,7 +40,12 @@ func main() {
 		log.Fatal(err)
 	}
 
-	srv := server.New(addr, rs)
+	cs, err := postgresql.NewComputedStatsService(10, 10)
```

Comment:
10, 10 (max idle connections, max open connections) is totally arbitrary.
If it does not seem appropriate values, feel free to suggest others! 
Or we could keep default ones and not set them anywhere in the code (default are 2 and 0 (meaning unlimited)).

Reply:
While looking for some info about it, I discovered `ConnMaxIdleTime` (max time a connection can be idled before being closed) and `ConnMaxLifetime` (max time a connection can be held before being closed).
To keep in mind if we also want some control over this too.